### ISSUE TITLE
Separate UI and REST

### ISSUE THREAD

**Issue by GRADeS-Gwen** [SORMAS-Foundation/SORMAS-Project#7794]
Hello,

We are using the "import" feature from the API/REST a lot, and it seem the it impact the usage of Sormas bu the "normal" users. When there is users and import simultanously, each of them is slower. So, I was wondering if we could separate the usage on 2 differents servers : one for "normal" usage and DB, and the other only for the REST calls. This way, the CPU usage by one case, do not impact the other.

But for this I need to know if there is some impacts and/or restrictions. The routing to REST or UI can easily be done through reverse proxy rules. But is their a problem if the 2 process write on the database simoultanously ? Is there a way to only start Rest and dependencies on the second server ?
In short, is it possible and how to make it possible ?

Thank you

**Comment by bernardsilenou** [comment]
@MartinWahnschaffe @MateStrysewske 
Any suggestion please?

**Comment by MateStrysewske** [comment]
I can't provide a sophisticated answer to this question because I'm not familiar enough with the technical details; but in theory, I'd say that if it works technically (i.e. both servers have access to the same database), I don't see a problem with this approach, as simultaneous database calls are already a thing right now anyway.

However, I'd definitely ask @MartinWahnschaffe and @StefanKock to provide their insights here.
